### PR TITLE
Remove invalid user preferences beta (no longer a beta, always enabled)

### LINKED ISSUE #4956: enable_new_user_preferences should be removed from galaxy.ini
Its good enough, isn't it?

### PR DESCRIPTION
**Pull request by dannon**
Fixes #4956, as well as cleaning up two other deprecated preferences and fixing several typos.

### CODE REVIEW

**Comment by martenson** [comment]
Lgtm. Maybe just for the peace of mind we should make sure to put this into release notes config changes. I do not think it warrants deprecation round though.

**Comment by dannon** [comment]
@martenson Yeah, the features corresponding to these entries are either already gone (sequencer communication) or integrated completely (toolform and preferences upgrades) and the entries in galaxy.ini are completely ignored.

File: config/galaxy.ini.sample [hunk 1]
```diff
@@ -1230,20 +1220,23 @@ use_interactive = True
...
+
 # Allow non-admin users to view available Data Manager options.
 #enable_data_manager_user_view = False
+
 # File where Data Managers are configured (.sample used if default does not
 # exist).
 #data_manager_config_file = config/data_manager_conf.xml
+
 # File where Tool Shed based Data Managers are configured.
 #shed_data_manager_config_file = config/shed_data_manager_conf.xml
```
Comment (nsoranzo) [inline config/galaxy.ini.sample:1236]: Add blank line here too?

Comment (dannon) [reply config/galaxy.ini.sample:1236]: @nsoranzo Thanks, missed that one in my reformatting!

**Review by nsoranzo — APPROVED** [review]

**Comment by guerler** [comment]
Thanks @dannon.

**Pull request merged**